pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -366,16 +370,39 @@ pub mod pallet {
 			currencies: BoundedVec<TokenMetaOf<T>, T::MaxCurrenciesPerPool>,
 			denomination: u8,
 			transferable: bool,
-			min_operation_balance: u128,
+			min_operation_balance: FungiblesBalanceOf<T>,
+			max_supply: FungiblesBalanceOf<T>,
```

Comment:
As in https://github.com/KILTprotocol/kilt-node/pull/875, this should probably be combined into a struct along with other currencies-wide settings.